### PR TITLE
integration tests for issue #1604

### PR DESCRIPTION
**Pull request by 121watts**
see issue #1604 

21 batch points, 3 tagRarity results in this test failing ~100% of the time.
This test seems to fail when a tag is associated with < 30% of total points.

### CODE REVIEW

File: cmd/influxd/server_integration_test.go [hunk 1]
```diff
@@ -419,3 +419,90 @@ func Test_Server5NodeIntegration(t *testing.T) {
...
+	createDatabase(t, testName, nodes, "foo")
+	createRetentionPolicy(t, testName, nodes, "foo", "bar")
+
+	bp := influxdb.BatchPoints{
+		Database: "foo",
+	}
+
+	measurementName := "my_measurement"
+
+	// 21 points, 3 tagRarity results in this test failing ~100% of the time
```
Comment (otoolep) [inline cmd/influxd/server_integration_test.go]: Also, when we do fix the issue, these comments will be inaccurate. We should just remove them, and not merge this test until the test passes.

Comment (otoolep) [inline cmd/influxd/server_integration_test.go]: By all means add this important information to the PR's comments, so we have a record of it.

**Comment by otoolep** [comment]
If this test takes more than a second or two to run, you should add the check for `short` that you see in the other tests, so developers can easily skip it. Ask @corylanou if you'd like some background on this.

**Comment by 121watts** [comment]
dealing with this in #1637

**Pull request closed without merging**